CMakeLists.txt

```diff
@@ -16,7 +16,9 @@ IF( WIN32 )
   set(BOOST_ALL_DYN_LINK OFF) # force dynamic linking for all libraries
 ENDIF(WIN32)
 
-FIND_PACKAGE(Boost 1.67 REQUIRED COMPONENTS unit_test_framework)
+if(NOT boost_headers_SOURCE_DIR)
```

Comment:
From cmake docs: `Regardless of the mode used, a <PackageName>_FOUND variable will be set to indicate whether the package was found.` I was expecting `boost_FOUND`.

Reply:
Checking for `boost_FOUND` does not work. I assume that this is set only by `find_package()`, and in our case boost is added via `add_subdirectory`.

Reply:
done, thanks Matt.